.github/workflows/release-cycle.yml

```
@@ -130,6 +130,11 @@ jobs:
         run: bash scripts/release/workflow/pack.sh
         env:
           PRERELEASE: ${{ needs.state.outputs.is_prerelease }}
+      - name: Upload tarball artifact
+        uses: actions/upload-artifact@v3
+        with:
+          name: ${{ format('{0}{1}', github.ref_name, '.tgz') }}
```

Isn't this preferable?

```suggestion
          name: ${{ github.ref_name }}.tgz
```